src/js/components/Menu/Menu.js

```diff
@@ -336,6 +337,7 @@ const Menu = forwardRef((props, ref) => {
           justify={item.justify}
           kind={!child ? 'option' : undefined}
           hoverIndicator={!child ? undefined : 'background'}
+          {...theme.menu.item}
```

Comment:
This looks like the theme would override `justify={item.justify}` above. I'mm wondering if we should more generally merge properties, via something like:
```
const defaultItemProps = { align: 'start' };
...
<Button {{ ...defaultItemProps, ...theme.menu.item, ...item }} \>
```
Does that make any sense, or am I missing something?

Reply:
Agreed.